### PR TITLE
Add draft-js-custom-styles definitions

### PR DESCRIPTION
**Pull request by megabayt**

### CODE REVIEW

File: types/draft-js-custom-styles/package.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
```
Comment (uniqueiniquity) [inline types/draft-js-custom-styles/package.json]: This file is unnecessary - please remove it.

File: types/draft-js-custom-styles/index.d.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+  [key: string]: {
+    add: (state: EditorState, value: CSSProperties) => EditorState;
+    remove: (state: EditorState, value: CSSProperties) => EditorState;
+    toggle: (state: EditorState, value: CSSProperties) => EditorState;
+    current: (state: EditorState) => CSSProperties[keyof CSSProperties];
+    styleFn: (state: EditorState, value: CSSProperties) => EditorState;
+  };
+}
+
+interface CustomStylesResult {
```
Comment (uniqueiniquity) [inline types/draft-js-custom-styles/index.d.ts]: Unless you explicitly don't want this exported, you can remove the `export {}` on line 35.

File: types/draft-js-custom-styles/tslint.json [hunk 3]
```diff
@@ -0,0 +1,5 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```
Comment (uniqueiniquity) [inline types/draft-js-custom-styles/tslint.json]: The `rules` property is unnecessary, please remove it.

File: types/draft-js-custom-styles/tsconfig.json [hunk 4]
```diff
@@ -0,0 +1,24 @@
+{
+    "compilerOptions": {
+        "module": "commonjs",
+        "lib": [
+            "es6",
+            "dom"
+        ],
+        "noImplicitAny": true,
+        "noImplicitThis": true,
+        "strictNullChecks": false,
```
Comment (uniqueiniquity) [inline types/draft-js-custom-styles/tsconfig.json]: Please enable `strictNullChecks`.

**Review by uniqueiniquity — CHANGES_REQUESTED** [review]

File: types/draft-js-custom-styles/tsconfig.json (hunk 4 above)
Comment (megabayt) [reply types/draft-js-custom-styles/tsconfig.json]: Done

File: types/draft-js-custom-styles/tslint.json (hunk 3 above)
Comment (megabayt) [reply types/draft-js-custom-styles/tslint.json]: Done

File: types/draft-js-custom-styles/index.d.ts (hunk 2 above)
Comment (megabayt) [reply types/draft-js-custom-styles/index.d.ts]: Done

**Pull request closed without merging**